schema.json

```diff
@@ -143,6 +143,15 @@
               "type": "string",
               "description": "e.g. Increased profits by 20% from 2011-2012 through viral advertising"
             }
+          },
+          "skills": {
+            "type": "array",
+            "description": "Specify skills used at the position",
+            "additionalItems": false,
+            "items": {
+              "type": "string",
+              "description": "e.g. sales, C#, JS, etc"
```

Comment:
Maybe give only one example on this line, to make clear that each array item contains only one skill, not one long string containing multiple skills. Just for clarity.

(Note that this PR jsonresume/resume-schema#409 is a duplicate of jsonresume/resume-schema#401.)